### ISSUE TITLE
EFS Volumes not getting mounted

### ISSUE THREAD

**Issue by jseiser** [terraform-aws-modules/terraform-aws-atlantis#367]
## Description

The EFS volume, is not getting mounted by the atlantis container.

- [x] ✋ I have searched the open/closed issues and my issue is not listed.

## Versions

- Module version [Required]: 4.0.0

- Terraform version: 1.5.2
- 
## Reproduction Code [Required]

https://github.com/terraform-aws-modules/terraform-aws-atlantis/blob/master/examples/github-separate/main.tf#L30

## Expected behavior

I would expect to have /home/atlantis mounted on the EFS volume

## Actual behavior

EFS volume is created, but nothing is mounting it to the container

## Additional context

```
  volume = { for k, v in merge(
    {
      name = "efs"
      efs_volume_configuration = {
        file_system_id     = module.efs.id
        transit_encryption = "ENABLED"
        authorization_config = {
          access_point_id = module.efs.access_points["atlantis"].id
          iam             = "ENABLED"
        }
      }
    },
    lookup(var.service, "volume", {})
  ) : k => v if var.enable_efs }
```

Render in JSON as

```
    "volumes": [
        {
            "name": "efs_volume_configuration",
            "host": {}
        },
        {
            "name": "name",
            "host": {}
        }
    ],
```

![AtlantisVolumes](https://github.com/terraform-aws-modules/terraform-aws-atlantis/assets/4855527/70dfecbc-f00b-477d-be27-37b97d46bfb7)

**Comment by bryantbiggs** [comment]
taking a look

**Comment by antonbabenko** [comment]
This issue has been resolved in version 4.0.1 :tada:
